symfony/client/src/orangehrmPimPlugin/components/EmployeeSubordinates.vue

```diff
@@ -0,0 +1,203 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div>
+    <save-supervisor
+      v-if="showSaveModal"
+      :http="http"
+      :reporting-methods="reportingMethods"
+      @close="onSaveModalClose"
+    ></save-supervisor>
+    <edit-supervisor
+      v-if="showEditModal"
+      :http="http"
+      :reporting-methods="reportingMethods"
+      @close="onEditModalClose"
+    ></edit-supervisor>
+    <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+      <profile-action-header @click="onClickAdd">
+        Assigned Subordinates
+      </profile-action-header>
+    </div>
+    <table-header
+      :selected="checkedItems.length"
+      :total="total"
+      :loading="isLoading"
+      @delete="onClickDeleteSelected"
+    ></table-header>
+    <div class="orangehrm-container">
+      <oxd-card-table
+        :headers="headers"
+        :items="items?.data"
+        :selectable="selectable"
```

Comment:
`oxd-card-table` now supports disabled prop. It accepts a boolean similar to `selectable`. Please use `disabled` prop instead of `selectable`. Refer [here](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmPimPlugin/pages/employee/EmployeeDependents.vue#L50).

Reply:
both props needed right? both serve different purposes isn't it?